pkg/package-server/provider/registry.go

```diff
@@ -189,9 +189,17 @@ func (p *RegistryProvider) syncCatalogSource(obj interface{}) (syncError error)
 		Namespace: source.GetNamespace(),
 		Name:      source.GetName(),
 	}
+
 	if sourceMeta := p.sources.GetMeta(key); sourceMeta != nil && sourceMeta.Address == address {
-		// If the address hasn't changed, don't bother creating a new source
-		logger.Debug("catalog address unchanged, skipping source creation")
```

Comment:
We still want to this debug line right? I thought my comment said it should be moved to the top.

Reply:
Saw your comment below. I'm fine with skipping this.